src/duckdb.js

```diff
@@ -130,7 +130,7 @@ export class DuckDBClient {
     await Promise.all(
       Object.entries(sources).map(async ([name, source]) => {
         if (source instanceof FileAttachment) { // bare file
-          await insertFile(db, name, source);
+          await insertFile(db, name, source, {}, config.untyped);
```

Comment:
E.g., here is one way to propagate the private untyped option:

```js
          await insertFile(db, name, source, {[untyped]: config[untyped]});
```

Reply:
Thank you, @mbostock! Appreciate this feedback. I believe I addressed your concerns, but since I needed the symbol in both table.js and duckdb.js, I defined it in table.js, and imported that definition in duckdb.js. Maybe there's a better way?